runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -169,10 +178,27 @@ else if (client.getGameState() != GameState.LOGGED_IN)
 		graphics.fill(new Rectangle(client.getCanvas().getSize()));
 		graphics.setColor(color);
 
-		if (Instant.now().minusMillis(FLASH_DURATION).isAfter(flashStart))
+		// Enforce a minimum flash duration of 2 seconds so as not to cancel it too quickly
```

Comment:
I would recommend against explicitly calling out the duration as 2 seconds, as that would become incorrect if `MINIMUM_FLASH_DURATION_MILLIS` were ever to change in the future.

Reply:
I also probably shouldn't have made the "Flash for 2 seconds" and "Solid for 2 seconds" options rely on MINIMUM_FLASH_DURATION_MILLIS to be 2000 so I've also reordered things a little.